mkdocs.yml

```diff
@@ -24,7 +24,6 @@ theme:
     - navigation.tracking
     - navigation.tabs
     - navigation.tabs.sticky
-    - toc.integrate
```

Comment:
I removed this because it was getting really cluttered with the deeply nested sections.